docs/BR.md

```diff
@@ -233,7 +234,7 @@ No stipulation.
 
 **Certificate Revocation List**: A regularly updated time-stamped list of revoked Certificates that is created and digitally signed by the CA that issued the Certificates.
 
-**Certification Authority**: An organization that is responsible for the creation, issuance, revocation, and management of Certificates. The term applies equally to both Roots CAs and Subordinate CAs.
+**Certification Authority**: A certificate generation service (people, procedures, systems and technology) that is trusted by one or more entities to create, sign, revoke, and provide status information for public key certificates and is operated by a Trust Service Provider.[Comment: The term "CA" encompasses people, too.]
```

Comment:
ETSI EN 319 411-1defines a *CA* as:

> A CA is commonly understood to be a type of Trust Service Provider (TSP), as defined in the Regulation (EU)
> No 910/2014 [i.14], and also a form of certification service provider as defined in the Electronic Signatures Directive
> 1999/93/EC [i.1], which issues public key certificates. However, in the present document the term is used more to
> reference the technical component of the TSP concerned with certificate issuance.
> The authority trusted by the users of the certification services (i.e. subscribers as well as relying parties) to assign
> certificates is the TSP. The TSP has overall responsibility for the provision of the certification services identified in
> clause 4.4. The CA, as well as the TSP, can be identified in the certificate as the issuer and its private key is used to sign certificates.

Reply:
Yeah, this is another spot where the BR usage and ETSI usage diverge.

In the BRs, a CA is an organizational unit, not a TSP.  TSPs are legal entities that operate CAs.